core/src/calls.rs

```diff
@@ -41,15 +41,15 @@ pub trait RpcNotification<T: Metadata>: Send + Sync + 'static {
 #[derive(Clone)]
 pub enum RemoteProcedure<T: Metadata> {
 	/// A method call
-	Method(Arc<RpcMethod<T>>),
+	Method(Arc<dyn RpcMethod<T>>),
```

Comment:
Is this really required now?

Reply:
Don't think so, just `cargo fix` being overzealous